Example/SectionedCollectionView/MyCustomData.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  MyCustomData.swift
+//  SectionedCollectionView_Example
+//
+//  Created by Esteban Arrua on 6/19/18.
+//  Copyright © 2018 CocoaPods. All rights reserved.
+//
+
+import Foundation
+import SectionedCollectionView
+
+public class MyCustomData: Selectable{
+    
+    public var name: String
+    
+    public init(name: String, selected: Bool) {
+        self.name = name
```

Comment:
why is the name assigned before the init and the selected after it? can we put them together?